src/components/TopNavbar.vue

```diff
@@ -43,6 +43,21 @@ const handleLogout = () => {
   authStore.logout();
   router.push('/');
 };
+
+const isAdmin = () => {
+  return authStore.roles.indexOf('SudoSOS - Board') !== -1;
```

Comment:
Magic string; ergens een enum doen lijkt me.

Reply:
Is gefixt in laatste commit.